docs/internal.md

```diff
@@ -0,0 +1,71 @@
+# Internal of Azure SignalR Service
+
+- [Server Connections](#server-connections)
+- [Client Connections](#client-connections)
+- [Transport Data between Client and Server](#transport)
+
+Azure SignalR Service is built on top of ASP.NET Core SignalR framework.
+You can start from a local ASP.NET Core SignalR application, then seamlessly move to Azure SignalR Service with a few lines of code change.
+
+The following diagram shows the typical architecture when you use Azure SignalR Service with your own application server.
+Behavior of clients and servers will be different from self-hosting ASP.NET Core SignalR.
+More details are explained in below sections.
+
+![Architecture](./images/arch.png)
+
+## Server Connections
+
+When self-hosting ASP.NET Core SignalR, application server will listen for client connections and serve them directly.
+
+With SignalR Service, application server will not accept SignalR client connections anymore.

```

Comment:
"the application server"

Reply:
Lets change this to say:

"With SignalR Service, the application server is no longer responsible for accepting persistent client connections."